.laminas-ci.json

```diff
@@ -1,5 +1,6 @@
-{
-    "ignore_php_platform_requirements": {
-        "8.2": true
-    }
-}
+{
+    "ignore_php_platform_requirements": {
+        "8.2": true,

```

Comment:
8.2 can be removed from ignore platform list

Reply:
Yes. I was wondering why PHP 8.2 was there. I will update.